src/pybind/rbd/rbd.pyx

```diff
@@ -898,13 +898,12 @@ class RBD(object):
     def trash_move(self, ioctx, name, delay=0):
         """
         Move an RBD image to the trash.
-
```

Comment:
I am not sure how is this causing an error *after* https://github.com/ceph/ceph/commit/89ace1b1e9bdd4bef35afb4cc89544b3ea6b1b5b 

There was no error after that fix went in, how is it that you are getting the "field list ends without a blank line" warning?